R/functions-Chromatograms.R

```diff
@@ -0,0 +1,104 @@
+## functions for Chromatograms class
+
+.validChromatograms <- function(x) {
+    msg <- character()
```

Comment:
adopted from Martin Morgan's *new* coding style - I'm trying to use wherever possible base functions. I'm however also fine with changing that to `Biobase::validMsg`.